### PR TITLE
0.2.0 cleaning script

### PR DESCRIPTION
**Pull request by martmull**

### CODE REVIEW

File: server/package.json [hunk 1]
```diff
@@ -85,6 +85,7 @@
     "graphql-yoga": "^4.0.4",
     "jest-mock-extended": "^3.0.4",
     "jsonwebtoken": "^9.0.0",
+    "lodash": "^4.17.21",
```
Comment (charlesBochet) [inline server/package.json:88]: we don't load the full loadash, only the functions that we need! (see examples below)

File: server/src/database/commands/clean-inactive-workspaces.command.ts [hunk 2]
```diff
@@ -4,20 +4,21 @@ import {
...
 import pipelinesSeed from 'src/core/pipeline/seed-data/sales-pipeline.json';
-import { arraysEqual } from 'src/utils/equal';
 import { WorkspaceService } from 'src/core/workspace/services/workspace.service';
 
 interface DataCleanInactiveOptions {
   days?: number;
   sameAsSeedDays?: number;
   dryRun?: boolean;
   confirmation?: boolean;
+  workspace?: string;
```
Comment (charlesBochet) [inline server/src/database/commands/clean-inactive-workspaces.command.ts:21]: workspaceId?

**Review by charlesBochet — APPROVED** [review]
Additional comments!

File: server/src/database/commands/clean-inactive-workspaces.command.ts (hunk 2 above)
Comment (martmull) [reply server/src/database/commands/clean-inactive-workspaces.command.ts:21]: of course 😁

**Pull request merged**